zebedee-cms/src/main/java/com/github/onsdigital/zebedee/api/Publish.java

```diff
@@ -67,6 +87,74 @@ public boolean publish(HttpServletRequest request, HttpServletResponse response)
         }
     }
 
+    /**
+     * Manual Publish the files in a collection
+     *
+     * @param collection       the collection to publish
+     * @param session          a session with editor priviledges
+     * @param skipVerification
+     * @return success
+     * @throws IOException
+     * @throws UnauthorizedException
+     * @throws BadRequestException
+     * @throws ConflictException     - If there
+     */
```

Comment:
Can you either fill out the params & exceptions fully OR just delete them from the javadoc. Im not that bothered about the params being documented but I'd rather they just weren't there rather than being blank

Reply:
I have made the function private as its only used internally. I have also removed the params and left the message.